state/txn.go

```diff
@@ -106,32 +99,17 @@ func (txn *Txn) getStateObject(addr types.Address) (*StateObject, bool) {
 		return obj.Copy(), true
 	}
 
-	data, ok := txn.snapshot.Get(txn.hashit(addr.Bytes()))
-	if !ok {
+	account, err := txn.snapshot.GetAccount(addr)
+	if err != nil {
```

Comment:
Could you add a log there?

Reply:
Similar to a previous comment by Evgeny. This function depends on arbitrary input from the smart contract, so if the account is not there (err != nil) is not an error. It is part of the workflow of the EVM. So if we were to add a log, there will be a smart contract combination that would print many logs on the screen.